File: vim/template/_WILDCARD_.rb
:silent %s/#CLASS#/\=ToCamel(RPath('%:t:r'))/g
:silent %s/#MODULE_DECL#/\=ModuleDeclaration()/g
:silent %s/#MODULE_DECL_END#/\=ModuleDeclarationEnd()/g
:silent %s/#I#/\=IndentationString(1)/g
:silent %s/#MI#/\=empty(ModuleString(RPath('%:h')))?'':IndentationString(1)/g
#!/usr/bin/env ruby
#MODULE_DECL#
#MI#class #CLASS#
#MI##I#def initialize
#MI##I##I##CURSOR#
#MI##I#end
#MI#end#MODULE_DECL_END#

File: vim/template/_WILDCARD_.py
#:%s/#CLASS#/\=ToCamel(RPath('%:t:r'))/g
#!/usr/bin/env python2
import unittest

class #CLASS#(object):

	def __init__(self):
		#CURSOR#

class Test#CLASS#(unittest.TestCase):

	def setUp(self):
		pass

	def tearDown(self):
		pass


if __name__ == "__main__":
	unittest.main()
